partcad-cli/src/partcad_cli/click/commands/convert.py

```
@@ -0,0 +1,138 @@
+@click.option(
+    "-i",
+    "--in-place",
+    help="Update the object's type in place after conversion",
+    is_flag=True,
+    default=False,
+)
+@click.argument("object", type=str, required=True)
+@click.pass_obj
+def cli(
+    ctx: object,
+    create_dirs: bool,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)